inc/saiacl.h

```diff
@@ -231,6 +231,9 @@ typedef enum _sai_acl_action_type_t
     /** Enable DTEL report for all packets without filtering (experimental) */
     SAI_ACL_ACTION_TYPE_DTEL_REPORT_ALL_PACKETS,
 
+    /** Enable NAT */
+    SAI_ACL_ACTION_TYPE_NAT,
+
```

Comment:
Is the only way to achieve NAT as per this proposal is to apply through ACL action?

Reply:
Please review the updated diffs